examples/terraform-submodules/gke-local/module.tf

```diff
@@ -14,7 +14,7 @@
 
 
 // Run:
-//  terraform apply [-var agones_version="1.3.0"]
+//  terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.1.0"]
```

Comment:
Why change the agones version to something older? If anything, what about using 1.4.0?

Reply:
I copy/pasted the previous command, but you are right, should update it to latest. Will do 👍